script/postrelease

```diff
@@ -38,5 +38,5 @@ verbose('npm run build')
 verbose('npm run build --app=meetingReadinessChecker')
 
 Dir.chdir('../serverless/')
-verbose('npm run deploy -- -b chime-sdk-demo-prod-canary -o chime-sdk-demo-prod-canary -s chime-sdk-demo-prod-canary -t -l')
+verbose('npm run deploy -- -b chime-sdk-demo-prod-canary -o chime-sdk-demo-prod-canary -s chime-sdk-demo-prod-canary -u false -t -l')
```

Comment:
Do you want to update the prod canary after the NPM release as well? This is just updating the one which will use the Chime client.